src/utils/plans.ts

```diff
@@ -12,29 +12,68 @@ export const ProductNameById = {
   [ProductId.SANBASE]: Product.SANBASE,
 } as const
 
-const nonDeprecatedFilter = ({ isDeprecated }: { isDeprecated: boolean }) => !isDeprecated
+export const nonDeprecatedFilter = ({ isDeprecated }: { isDeprecated: boolean }) => !isDeprecated
 export const keepNonDeprecatedPlans = (plans: SAN.Plan[]) => plans.filter(nonDeprecatedFilter)
 
 export const checkIsSanbaseProduct = (product: Pick<SAN.Product, 'id'>) =>
   +product.id === ProductId.SANBASE
 
+export const checkIsSanApiProduct = (product: Pick<SAN.Product, 'id'>) =>
+  +product.id === ProductId.SANAPI
+
 export enum Plan {
+  //deprecated
+  PRO_PLUS = 'PRO_PLUS',
+
   FREE = 'FREE',
   PRO = 'PRO',
-  PRO_PLUS = 'PRO_PLUS',
+  MAX = 'MAX',
+
+  BUSINESS_PRO = 'BUSINESS_PRO',
+  BUSINESS_MAX = 'BUSINESS_MAX',
+  CUSTOM = 'CUSTOM',
+}
+
+export const INDIVIDUAL_PLANS = [Plan.FREE, Plan.PRO, Plan.MAX] as const
+export const BUSINESS_PLANS = [Plan.BUSINESS_PRO, Plan.BUSINESS_MAX, Plan.CUSTOM] as const
+
+export const individualPlanMap = arrToMap(INDIVIDUAL_PLANS)
+export const businessPlansMap = arrToMap(BUSINESS_PLANS)
+
+function arrToMap<T extends string | number | symbol>(arr: readonly T[]) {
+  return arr.reduce(
+    (prev, plan) => ({ ...prev, [plan]: true }),
+    {} as Record<(typeof arr)[number], true>,
+  )
 }
 
 export const PlanName = {
-  [Plan.FREE]: 'Free',
-  [Plan.PRO]: 'Pro',
   [Plan.PRO_PLUS]: 'Pro+',
+
+  [Plan.FREE]: 'FREE',
+  [Plan.PRO]: 'Sanbase Pro',
+  [Plan.MAX]: 'Sanbase Max',
+
+  [Plan.BUSINESS_PRO]: 'Business Pro',
+  [Plan.BUSINESS_MAX]: 'Business Max',
+  [Plan.CUSTOM]: 'Enterprise',
 } as const
 
 export enum Billing {
   MONTH = 'month',
   YEAR = 'year',
 }
 
+export enum PlanType {
+  INDIVIDUAL = 'individual',
+  BUSINESS = 'business',
+}
+
+export const isIndividualPlan = ({ name }: SAN.Plan) => !!individualPlanMap[name]
+export const isBusinessPlan = ({ name }: SAN.Plan) => !!businessPlansMap[name]
+
+export const isPlanWithPrice = ({ amount }: SAN.Plan) => amount > 0
+
 export const checkIsYearlyPlan = ({ interval }: Pick<SAN.Plan, 'interval'>) =>
```

Comment:
Functions that return boolean are prefixed with a `check` verb -> `checkIsIndividualPlan`.
`isIndividualPlan` indicates a variable that stores a boolean value.